src/js/shoppingCart.mjs

```diff
@@ -42,4 +50,28 @@ function calculateTotal(cart) {
   });
   document.querySelector(".cart-footer").classList.remove("hide");
   return total;
+}
+
+function addClickEvents(cart) {
+  cart.forEach((cartItem) => {
+    document
+    .getElementById(cartItem.Id)
+    .addEventListener("click", removeItem);
+  });
+}
+
+async function removeItem(e) {
```

Comment:
I feel like the add to cart and remove from cart functions should be together, but it felt awkward putting this function in the productDetails.mjs file. Any recommendations on where to put these two functions?

Reply:
The remove cart items function would only be used on the cart page, so it should go in the shoppingCart.mjs file. We may end up moving addToCart anyway because we will eventually need to use it on the product preview models on the home page and product list pages.